src/components/StatsBoxGrid/GridItem.tsx

```diff
@@ -1,38 +1,36 @@
-import React from "react"
 import { kebabCase } from "lodash"
 import { MdInfoOutline } from "react-icons/md"
 import { Area, AreaChart, ResponsiveContainer, XAxis, YAxis } from "recharts"
-import { Box, Flex, Icon, Text,VStack } from "@chakra-ui/react"
+import { Box, Flex, Icon, Text, VStack } from "@chakra-ui/react"
+
+import type { StatsBoxMetric } from "@/lib/types"
+
+import { RANGES } from "@/lib/constants"
 
-import { Direction } from "../../types"
 import InlineLink from "../Link"
 import OldText from "../OldText"
 import StatErrorMessage from "../StatErrorMessage"
-import StatLoadingMessage from "../StatLoadingMessage"
 import Tooltip from "../Tooltip"
 import Translation from "../Translation"
 
-import { Metric, ranges } from "./useStatsBoxGrid"
-
-const tooltipContent = (metric: Metric) => (
+const tooltipContent = (metric: StatsBoxMetric) => (
   <div>
     <Translation id="data-provided-by" />{" "}
     <InlineLink to={metric.apiUrl}>{metric.apiProvider}</InlineLink>
   </div>
 )
 
-interface IGridItemProps {
-  metric: Metric
-  dir?: Direction
```

Comment:
Remove `dir`; managed by top-level `html` attribute now

Reply:
@wackerow should `dir` be removed from [`HomePage`](https://github.com/ethereum/ethereum-org-fork/blob/6b7925a1a5ec7a7b0c07b6e82209a78b4d08c9ac/src/pages/index.tsx#L226C9-L226C12) then?